components/script/dom/bindings/root.rs

```diff
@@ -275,6 +275,7 @@ impl RootCollection {
     unsafe fn unroot(&self, object: *const dyn JSTraceable) {
         assert_in_script();
         let roots = &mut *self.roots.get();
+        #[allow(clippy::vtable_address_comparisons)]
```

Comment:
Based on https://github.com/rust-lang/rust-clippy/issues/6524, it seems that this lint will be removed in a new version of Rust. I don't think we want to hide this warning if it will be removed soon anyway. Maybe the thing to do here is to leave it for now and then see if upgrading our Rust version fixes it.

Reply:
@mrobinson makes sense! removed this change in the latest commit